src/components/SwipeButtons.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { View, Text, StyleSheet, TouchableOpacity, Button } from 'react-native'
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+import { Feather } from '@expo/vector-icons';
+
+export type SwipeButtonsParamList = {
```

Comment:
Can we eventually move this to the types.tsx file?

Maybe it's still here because this component is still in development?